CrossRef.js

```diff
@@ -410,14 +307,52 @@ var testCases = [
 				"bookTitle": "Heimat, Region, and Empire",
 				"ISBN": "9780230391116",
 				"publisher": "Palgrave Macmillan",
-				"language": "en",
 				"date": "2012-10-17",
 				"DOI": "10.1017/CCOL0521858429.016",
```

Comment:
That's quite confusing, but it's actually just a dirty test. I had to hand-craft these, since Scaffold doesn't make tests for search translators (yet). I was copy-pasting the first test and then adjusting based on Translator Tester output, which doesn't pick up on invalid fields ( we don't have DOI for bookSection).. Will fix it.